spec/helpers.rb

```diff
@@ -50,13 +50,25 @@ def create_pending_vm(template, name, token = nil)
   redis.hset("vmpooler__vm__#{name}", "template", template)
 end
 
-def create_vm(name, token = nil)
+def create_vm(name, token = nil, r = nil)
+  redis = r if r
```

Comment:
This is a bit tricky (I think we're shadowing a method with a local variable?).  Might be less surprising as:

``` ruby
def create_vm(name, token = nil, redis_handle = nil)
  redis_db = redis_handle ? redis_handle : redis
  redis_db.hset("vmpooler__vm__#{name}", 'checkout', Time.now)
  redis_db.hset("vmpooler__vm__#{name}", 'token:token', token) if token
end
```